src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { FC } from 'react';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tabId: string) => void,
+}
+
+export const Tabs: FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const selectTabId = (tab: Tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab.id);
```

Comment:
let's pass whole `tab` in `onTabSelected`, it will allow us to remove `selectedTab` variable in `App.tsx`. 
(in this case, you need to rename your state to selectedTab and setSelectedTab)